src/components/bread-crumb/GroupBreadCrumbs.tsx

```diff
@@ -42,7 +43,10 @@ export const GroupBreadCrumbs = () => {
                   {group.name}
                 </Link>
               )}
-              {subGroups.length - 1 === i && <>{t("groups:groupDetails")}</>}
+              {isLastGroup(i) && group.id !== "search" && (
+                <>{t("groups:groupDetails")}</>
+              )}
+              {isLastGroup(i) && group.id === "search" && <>{group.name}</>}
```

Comment:
There is no need to use a fragment here, as a string is also a valid JSX element. We can also use a ternary operation to reduce the amount of calls to `isLastGroup()`.

```suggestion
              {isLastGroup(i) &&
                (group.id === "search" ? group.name : t("groups:groupDetails"))}
```